tutorials/abap-environment-abapgit/abap-environment-abapgit.md

```diff
@@ -1,7 +1,7 @@
 ---
 auto_validation: true
 title: Use abapGit to Transfer Your On-Premise ABAP Source Code to the Cloud
-description: Transfer Your ABAP Source Code from your on-premise SAP system to your SAP Cloud Platform ABAP Environment instance over a Github Repository using the abapGit report and the ADT plugin for abapGit.
+description: Transfer Your ABAP Source Code from your on-premise SAP system to your SAP BTP ABAP Environment instance over a Github Repository using the abapGit report and the ADT plugin for abapGit.
```

Comment:
Please use the full name "SAP Business Technology Platform" at least once at the beginning of your tutorial before you use the abbreviation.